indexing-service/src/main/java/org/apache/druid/indexing/worker/config/WorkerConfig.java

```diff
@@ -29,6 +29,7 @@
 
 /**
  */
+@SuppressWarnings("SS_SHOULD_BE_STATIC")
```

Comment:
This doesn't seem to be working, travis is still failing with the same error https://travis-ci.org/apache/incubator-druid/jobs/562722959#L1985:
```
[INFO] Total bugs: 3
[ERROR] Unread field: org.apache.druid.indexing.worker.config.WorkerConfig.intermediaryPartitionCleanupPeriodSec; should this field be static? [org.apache.druid.indexing.worker.config.WorkerConfig] At WorkerConfig.java:[line 50] SS_SHOULD_BE_STATIC
[ERROR] Unread field: org.apache.druid.indexing.worker.config.WorkerConfig.intermediaryPartitionDiscoveryPeriodSec; should this field be static? [org.apache.druid.indexing.worker.config.WorkerConfig] At WorkerConfig.java:[line 47] SS_SHOULD_BE_STATIC
[ERROR] Unread field: org.apache.druid.indexing.worker.config.WorkerConfig.version; should this field be static? [org.apache.druid.indexing.worker.config.WorkerConfig] At WorkerConfig.java:[line 39] SS_SHOULD_BE_STATIC
```

Reply:
Please use `@SuppressFBWarnings` instead.

Reply:
My mistake, had only a couple of minutes to squeeze out the PR ;)